packages/repo-tools/src/commands/api-reports/api-extractor.ts

```diff
@@ -393,19 +393,16 @@ export async function runApiExtraction({
     );
 
     const remainingReportFiles = new Set(
-      fs
-        .readdirSync(projectFolder)
-        .filter(
-          filename =>
-            filename.match(/^(.+)-api-report\.md$/) ||
-            filename.match(/^api-report(-.+)?\.md$/),
-        ),
+      fs.readdirSync(projectFolder).filter(filename =>
+        // https://regex101.com/r/QDZIV0/1
+        filename.match(/^.*?api-report(-[^.-]+)?(\.md(\.api\.md)?)$/),
+      ),
     );
 
     for (const packageEntryPoint of packageEntryPoints) {
       const suffix =
         packageEntryPoint.name === 'index' ? '' : `-${packageEntryPoint.name}`;
-      const reportFileName = `api-report${suffix}.md`;
+      const reportFileName = `api-report${suffix}`;
```

Comment:
Looks like API extractor has moved away from the `api-report` naming in general, using the `.api.md` extension to identify these files instead. I do think we should switch to `.api.md`, but it needs a bit more thinking about how we could best apply it on our side. Some thoughts on that:

- I like `.api.md` as a way to identify these files reliably, without the need for matching on the rest of the name etc.
- I think we should use `.cli.md` to in a similar way identify our own CLI reports, which is a thing that's unique to the `repo-tools` implementation.
- I like having these files gathered in one place and in some sensible order. My first hunch here was to rename them all to `<name>.api.md` where name is the entry point name, or `index` for the main entry point. Thinking more about it I don't like this too much, since the files with be spread across the package directly in a quite unpredictable way. I'd say that leaves us with two options:
  1. Find (or keep) a filename prefix, for example sticking to `api-report*`, or shortening to just `api*` or `report*`, or something completely new like `exports*`.
  2. Place all of these files in a subdirectory, for example `reports`

This is perhaps not a super critical decision but I feel it needs a bit of thinking so that we can minimize the changes. @backstage/maintainers

Reply:
I think `report.api.md` for the main export and a suffix like `report-client.api.md` is the sweet spot in regard human readability and easily parseable format. 

We have: 
- all reports together in the file directory
- does not duplicate information like `api-report.api.md` 
- sub exports are easy detectable and retrievable as the dash only exists if there is a sub export. 
- the standard file extension is used.

Reply:
Yep I like it 👍 

are you happy to switch the PR over to that?